From 60d719fe9105431a0d5361dbd54db20b6eba7234 Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Tue, 30 Sep 2014 09:10:09 +0200
Subject: Add libedbus-private.la into LIBADD where needed

This fixes a build break on some systems (for example Debian)
after changes in commit a2790163af4d3f375a778055d0e2699207dfd050.

diff --git a/addressbook/libebook/Makefile.am b/addressbook/libebook/Makefile.am
index c70addb..1029495 100644
--- a/addressbook/libebook/Makefile.am
+++ b/addressbook/libebook/Makefile.am
@@ -59,6 +59,7 @@ libebook_1_2_la_LIBADD = \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/addressbook/libedata-book/libedata-book-1.2.la \
+	$(top_builddir)/private/libedbus-private.la \
 	$(EVOLUTION_ADDRESSBOOK_LIBS) \
 	$(CAMEL_LIBS) \
 	$(NULL)
diff --git a/addressbook/libedata-book/Makefile.am b/addressbook/libedata-book/Makefile.am
index 8e56a52..a15e18a 100644
--- a/addressbook/libedata-book/Makefile.am
+++ b/addressbook/libedata-book/Makefile.am
@@ -51,6 +51,7 @@ libedata_book_1_2_la_LIBADD = \
 	$(top_builddir)/addressbook/libegdbus/libegdbus-book.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
+	$(top_builddir)/private/libedbus-private.la \
 	$(DB_LIBS) \
 	$(CAMEL_LIBS) \
 	$(SQLITE3_LIBS) \
@@ -127,6 +128,7 @@ evolution_addressbook_factory_subprocess_SOURCES = \
 evolution_addressbook_factory_subprocess_LDADD = \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
+	$(top_builddir)/private/libedbus-private.la \
 	libedata-book-1.2.la \
 	$(EVOLUTION_ADDRESSBOOK_LIBS) \
 	$(LIBSECRET_LIBS) \
diff --git a/calendar/libecal/Makefile.am b/calendar/libecal/Makefile.am
index eb12c37..e2dd6b8 100644
--- a/calendar/libecal/Makefile.am
+++ b/calendar/libecal/Makefile.am
@@ -62,6 +62,7 @@ libecal_1_2_la_SOURCES = \
 libecal_1_2_la_LIBADD = \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/calendar/libegdbus/libegdbus-cal.la \
+	$(top_builddir)/private/libedbus-private.la \
 	$(EVOLUTION_CALENDAR_LIBS) \
 	$(CAMEL_LIBS) \
 	$(NULL)
diff --git a/calendar/libedata-cal/Makefile.am b/calendar/libedata-cal/Makefile.am
index 98d19d2..3986e0f 100644
--- a/calendar/libedata-cal/Makefile.am
+++ b/calendar/libedata-cal/Makefile.am
@@ -40,6 +40,7 @@ libedata_cal_1_2_la_LIBADD = \
 	$(top_builddir)/calendar/libegdbus/libegdbus-cal.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
+	$(top_builddir)/private/libedbus-private.la \
 	$(EVOLUTION_CALENDAR_LIBS) \
 	$(CAMEL_LIBS) \
 	$(NULL)
@@ -99,6 +100,7 @@ evolution_calendar_factory_subprocess_SOURCES = \
 evolution_calendar_factory_subprocess_LDADD = \
 	$(top_builddir)/libebackend/libebackend-1.2.la \
 	$(top_builddir)/libedataserver/libedataserver-1.2.la \
+	$(top_builddir)/private/libedbus-private.la \
 	libedata-cal-1.2.la \
 	$(EVOLUTION_CALENDAR_LIBS) \
 	$(LIBSECRET_LIBS) \
-- 
cgit v0.10.1

